t/24-worker-engine.t

```diff
@@ -71,4 +72,12 @@ delete($settings->{NUMDISKS});
 $got = OpenQA::Worker::Engines::isotovideo::detect_asset_keys($settings);
 is_deeply($got, $expected, 'Asset settings are correct (no UEFI or NUMDISKS)') or diag explain $got;
 
+subtest 'caching' => sub {
+    is(OpenQA::Worker::Engines::isotovideo::cache_assets, undef, 'cache_assets has nothing to do without assets');
+    my %assets = (
+        ISO => 'foo.iso',
+    );
+    my $cache_client = Test::MockModule->new('OpenQA::Worker::Cache::Client');
```

Comment:
Should there be a check to see if `cache_assets` is not `undef`? The test looks very incomplete. Or I might not be clear what it's trying to assert.

Reply:
Please see the  commit description in https://github.com/os-autoinst/openQA/pull/2116/commits/522b24fe96a5a97487e1b46a87195c2586894af8 . The test is really just trying to "call code and see if it explodes". Real tests should come with https://github.com/os-autoinst/openQA/pull/1962 or a follow-up

Reply:
I don't get it either i'm afraid. Why mock `OpenQA::Worker::Cache::Client` here?

Reply:
ok, fine. I will remove the test again from this PR and focus on the output format only (tested manually). Then we can discuss that test-issue further in https://github.com/os-autoinst/openQA/pull/1962 or maybe then it is already resolved with more tests.